ci/tasks/send-job-release-info-to-hosted-graphite.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+    tag: 3.16
+params:
+  BUILD_PIPELINE_NAME:
+  BUILD_JOB_NAME:
+  RELEASE_NUMBER:
+  HOSTED_GRAPHITE_API_TOKEN:
+run:
+  path: sh
+  args:
+    - -ec
+    - |
+      RELEASE=$(echo "${RELEASE_NUMBER}" | cut -d'-' -f 1)
+      METRIC_DATA="${HOSTED_GRAPHITE_API_TOKEN}.ci.concourse.$BUILD_PIPELINE_NAME.$BUILD_JOB_NAME.releaseNumber $RELEASE"
```

Comment:
A couple of suggestions 

- try with curly braces around the `BUILD_PIPELINE_NAME` and `BUILD_JOB_NAME`
- replace the `releaseNumber` string at the end with something more descriptive for our metric, like `jobStarted` maybe?